src/java.base/share/classes/java/io/StreamTokenizer.java

```diff
@@ -628,7 +629,7 @@ else if ('0' <= c && c <= '9') {
             peekc = c;
             sval = String.copyValueOf(buf, 0, i);
             if (forceLower)
-                sval = sval.toLowerCase();
+                sval = sval.toLowerCase(Locale.ROOT);
```

Comment:
This one gave me pause. AFAICS it's probably OK - but it might warant a CSR and an update of the specification to explicitly state how the lower case conversion is performed (update the `lowerCaseMode` method spec). Could we handle that in a separate PR?